keyboards/hineybush/h87_g2/chconf.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2024 Josh Hinnebusch
```

Comment:
Is this file actually required?

Reply:
not sure, everything builds fine without so I'll remove it.